ts-client/src/dlmm/helpers/index.ts

```diff
@@ -197,8 +198,70 @@ export async function chunkedGetMultipleAccountInfos(
   return accountInfos;
 }
 
-export const computeBudgetIx = () => {
-  return ComputeBudgetProgram.setComputeUnitLimit({
-    units: 1_400_000,
+/**
+ * Gets the estimated compute unit usage with a buffer.
+ * @param connection A Solana connection object.
+ * @param buffer The buffer to add to the estimated compute unit usage. Max value is 1. Default value is 0.1 if not provided.
+ * @returns The estimated compute unit usage with the buffer.
+ */
+export const getEstimatedComputeUnitUsageWithBuffer = async (
+  connection: Connection,
+  instructions: TransactionInstruction[],
+  feePayer: PublicKey,
+  buffer?: number
+) => {
+  if (!buffer) {
+    buffer = 0.1;
```

Comment:
I think the surplus should be between 50k-200k 
the additional CU = (estimatedComputeUnitUsage * buffer).max(50k).min(200k)

Reply:
Fixed